Description: Disable geo-specific search defaults for users in the US, as
 these override the locale-specific defaults even on partner builds
Author: Chris Coulson <chris.coulson@canonical.com>
Bug-Ubuntu: https://launchpad.net/bugs/1485741
Forwarded: no

--- a/l10n/be/browser/firefox-l10n.js
+++ b/l10n/be/browser/firefox-l10n.js
@@ -5,7 +5,3 @@
 #filter substitution
 
 pref("browser.fixup.alternate.suffix", ".by");
-
-# LOCALIZATION NOTE: this preference is set to true only for a specific set of
-# locales, without this line the setting is set to false by default.
-pref("browser.search.geoSpecificDefaults", true);
--- a/l10n/kk/browser/firefox-l10n.js
+++ b/l10n/kk/browser/firefox-l10n.js
@@ -3,8 +3,3 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 #filter substitution
-
-# LOCALIZATION NOTE: this preference is set to true for en-US specifically,
-# locales without this line have the setting set to false by default.
-pref("browser.search.geoSpecificDefaults", true);
-
--- a/l10n/ru/browser/firefox-l10n.js
+++ b/l10n/ru/browser/firefox-l10n.js
@@ -3,8 +3,3 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 #filter substitution
-
-
-# LOCALIZATION NOTE: this preference is set to true only for a specific set of
-# locales, without this line the setting is set to false by default.
-pref("browser.search.geoSpecificDefaults", true);
--- a/l10n/tr/browser/firefox-l10n.js
+++ b/l10n/tr/browser/firefox-l10n.js
@@ -3,8 +3,3 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 #filter substitution
-
-# LOCALIZATION NOTE: this preference is set to true for en-US specifically,
-# locales without this line have the setting set to false by default.
-pref("browser.search.geoSpecificDefaults", true);
-
--- a/l10n/uk/browser/firefox-l10n.js
+++ b/l10n/uk/browser/firefox-l10n.js
@@ -3,8 +3,3 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 #filter substitution
-
-
-# LOCALIZATION NOTE: this preference is set to true only for a specific set of
-# locales, without this line the setting is set to false by default.
-pref("browser.search.geoSpecificDefaults", true);
--- a/l10n/zh-TW/browser/firefox-l10n.js
+++ b/l10n/zh-TW/browser/firefox-l10n.js
@@ -4,10 +4,6 @@
 
 #filter substitution
 
-# LOCALIZATION NOTE: this preference is set to true for en-US specifically,
-# locales without this line have the setting set to false by default.
-pref("browser.search.geoSpecificDefaults", true);
-
 
 #ifdef XP_WIN
 
